doc/md/writing-motoko/pattern-matching.md

```diff
@@ -28,48 +57,40 @@ switch ("Adrienne", #female) {
-| wildcard                   | `_`                             | everywhere | no                                    |                                          |
-| typed                      | `age : Nat`                     | everywhere | conditional                           |                                          |
-| option                     | `?0`, `?val`                    | everywhere | yes                                   |                                          |
-| tuple                      | `( component0, component1, …​ )` | everywhere | conditional                           | must have at least two components        |
-| object                     | `{ fieldA; fieldB; …​ }`         | everywhere | conditional                           | allowed to mention a subset of fields    |
-| field                      | `age`, `count = 0`              | object     | conditional                           | `age` is short for `age = age`           |
-| variant                    | `#celsius deg`, `#sunday`       | everywhere | yes                                   | `#sunday` is short form for `#sunday ()` |
-| alternative (`or`-pattern) | `0 or 1`                        | everywhere | depends                               | no alternative may bind an identifier    |
-
-The following table summarises the different ways of pattern matching.
+The last kind of pattern is the `or` pattern. As its name suggests, these are two or more patterns that are separated by the keyword `or`. Each of the sub-patterns must bind to the same set of identifiers, and is matched from left-to-right. An `or` pattern fails when its rightmost sub-pattern fails.
 
 ## Additional information about about patterns
 
 Since pattern matching has a rich history and interesting mechanics, a few additional comments are justified.
 
-terminology  
-The (usually structured) expression that is being matched is frequently called the *scrutinee* and the patterns appearing behind the keyword `case` are the *alternatives*. When every possible scrutinee is matched by (at least one) alternative, then we say that the scrutinee is *covered*. The patterns are tried in top-down fashion and thus in case of *overlapping* patterns the one higher-up is selected. An alternative is considered *dead* (or *inactive*), if for every value that it matches there is higher-up alternative that is also matched.
+### Terminology
+
+The usually structured expression that is being matched is frequently called the **scrutinee**. The patterns appearing behind the keyword `case` are the **alternatives**. When every possible scrutinee is matched by at least one alternative, then we say that the scrutinee is **covered**. The patterns are tried in top-down fashion and thus in case of overlapping patterns, the one higher-up is selected. An alternative is considered dead or inactive if for every value that it matches there is higher-up alternative that is also matched.
```

Comment:
```suggestion

The expression whose value is being matched is frequently called the **scrutinee**. The patterns appearing behind the keyword `case` are the **alternatives**. When every possible value of the scrutinee is matched by at least one alternative, then we say that the scrutinee is **covered**. The alternatives are tried in order; in case of overlapping patterns, the earlier one is selected. An alternative is considered dead or redundant if for every value that it matches there is already some earlier alternative that also matches the value.
```